backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/quickinput/orderline/OrderLineQuickInputDescriptorFactory.java

```diff
@@ -175,7 +175,7 @@ private ProductLookupDescriptor createProductLookupDescriptor(@NonNull final Opt
 						.setCtxTableName(null) // ctxTableName
 						.setCtxColumnName(IOrderLineQuickInput.COLUMNNAME_M_HU_PI_Item_Product_ID)
 						.setDisplayType(DisplayType.TableDir)
-						.setAD_Val_Rule_ID(540365) // FIXME: hardcoded "M_HU_PI_Item_Product_For_Org_and_Product_and_DatePromised"
+						.setAD_Val_Rule_ID(540365)// FIXME: hardcoded "M_HU_PI_Item_Product_For_Org_and_Product_and_DatePromised"
```

Comment:
i didn't understand in the concept, why this was supposed to be changed..so i guess it's fine that it effectively was *not* changed at all